src/Features/Core/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -237,9 +237,14 @@ private IEnumerable<INamespaceSymbol> GetNamespacesForMatchingTypesAsync(Semanti
 
             var expression = node.Parent;
 
-            var symbols = await GetSymbolsAsync(project, diagnostic, node, semanticModel, syntaxFacts, cancellationToken).ConfigureAwait(false);
+            var symbols = await GetSymbolsAsync(project, node, semanticModel, syntaxFacts, cancellationToken).ConfigureAwait(false);
 
-            return FilterForExtensionMethods(semanticModel, namespacesInScope, syntaxFacts, expression, symbols, cancellationToken);
+            if (symbols != null)
```

Comment:
The code could be more compact and arguably no less readable with a ?: operator.